task1/style.css

```diff
@@ -0,0 +1,77 @@
+body {
+    background-image: url('https://htmlcolorcodes.com/assets/images/html-color-codes-color-palette-generators-7a5b8241.jpg');
+    background-repeat: repeat;
+    text-align: center;
+    font-family: Roboto, sans-serif;
+}
+
+ul {
+    list-style-type: none;
+    top: 0;
+    left: 0;
+    margin: 0;
+    padding: 0;
+    position: fixed;
+    width: 100%;
+    background-color: black;
+    /* opacity: 0.7; */
```

Comment:
remove commented code from pr, master should not contain commented code, unless it necessary comments.